powered-by.html

```diff
@@ -412,7 +412,22 @@
             "logo": "rabobank.jpg",
             "logoBgColor": "#ffffff",
             "description": "Rabobank is one of the 3 largest banks in the Netherlands. Its digital nervous system, the Business Event Bus, is powered by Apache Kafka and Kafka Streams."
-        }
+        },{
+    "link":  "http://www.portoseguro.com.br/",
```

Comment:
Nit: can we make indentation match the above for consistency? If we don't do this we tend to get people's editors reformatting things which adds noise to the diffs